demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -513,8 +514,6 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
         (buttonMeetingLeave as HTMLButtonElement).disabled = true;
         this.leave();
         (buttonMeetingLeave as HTMLButtonElement).disabled = false;
-        // @ts-ignore
-        window.location = window.location.pathname;
```

Comment:
Is this being removed to ensure the logs go in even after the meeting leave/end button is clicked ?

Reply:
Yes, we are handling redirection in the `audioVideoDidStop`.